artifactory/manager.go

```diff
@@ -61,6 +48,23 @@ func NewWithProgress(config config.Config, progress ioutils.ProgressMgr) (Artifa
 	return manager, err
 }
 
+func BuildJFrogHttpClient(config config.Config, authDetails auth.ServiceDetails) (*jfroghttpclient.JfrogHttpClient, error) {
```

Comment:
(Unless you're planning to use it outside of this package)
```suggestion
func buildJFrogHttpClient(config config.Config, authDetails auth.ServiceDetails) (*jfroghttpclient.JfrogHttpClient, error) {
```

Reply:
Should use it outside :)